grails-app/views/dataResource/show.gsp

```diff
@@ -21,6 +21,12 @@
               height: 170px;
               float: right;
             }
+
+            .privateMenuLink {
```

Comment:
Css in a template?

Reply:
this css already was here, so moved that class here